devito/propagator.py

```diff
@@ -165,6 +181,10 @@ def cfunction(self):
         return self._cfunction
 
     @property
+    def timings(self):
+        return self.profiler.as_dictionary
```

Comment:
Accessing this property fails if we ran with `profile=False`. Can we make sure this is handled more gracefully, eg. return `None` or similar.